src/Orleans.Core.Abstractions/IDs/GrainId.cs

```diff
@@ -48,6 +48,11 @@ internal static GrainId GetGrainId(UniqueKey key)
             return FindOrCreateGrainId(key);
         }
 
+        internal static GrainId GetSystemGrainId(Guid guid)
```

Comment:
If I am not mistaken, it will introduce a breaking change in Abstaction, and we would like to avoid that... Could we move this helper method somewhere else?

Reply:
That is just partial revert of changes done at https://github.com/dotnet/orleans/pull/4479/files#diff-6f8ed860e2abcd355cb00107fd1653b2L51.

Reply:
Sorry I missed that!